run.cc

```diff
@@ -3,80 +3,180 @@
 #include <fstream>
```

Comment:
We should totally rename this file to `run_mcmc.cc`, by the way

Reply:
Ok to me